build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/tscTask.ts

```diff
@@ -466,6 +466,30 @@ export abstract class TscDependentTask extends LeafWithDoneFileTask {
 	protected abstract getToolVersion(): Promise<string>;
 }
 
+interface TscMultiConfig {
+	targets: {
+		extName?: string;
+		packageOverrides?: Record<string, unknown>;
+	}[];
+	projects: string[];
+}
+
+// This function is taken from tsc-multi.
+function configKeyForPackageOverrides(overrides: Record<string, unknown> | undefined) {
+	if (overrides === undefined) return "";
+
+	const str = JSON.stringify(overrides);
+	if (str.length === 0) return "";
```

Comment:
Nit: I don't believe JSON.stringify() to returns empty string.

The closest is 'JSON.stringify(undefined) -> undefined', which is not a string (and already handled above).

Other inputs either produce a non-empty string or throw an error (e.g., type error, circular references, etc.)

null -> 'null', "" -> '""', etc.